src/main/kotlin/org/amshove/kluent/Basic.kt

```diff
@@ -20,3 +20,20 @@ infix fun Any?.shouldBeInstanceOf(className: Class<*>) = this `should be instanc
 infix fun Any?.`should not be instance of`(className: Class<*>) = assertFalse("Expected $this to not be an instance of $className", className.isInstance(this))
 infix fun Any?.shouldNotBeInstanceOf(className: Class<*>) = this `should not be instance of` className
 
+fun Any?.`should be null`() = assertNull(this)
+fun Any?.shouldBeNull() = this.`should be null`()
+
+fun Any?.`should not be null`() = assertNotNull(this)
```

Comment:
how about `assertNotNull("shouldn't be null", this)` or something similar?
It could be a lot more descriptive than
```
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertNotNull(Assert.java:712)
	at org.junit.Assert.assertNotNull(Assert.java:722)
```

Reply:
Good find, I've opened an issue #88 and will fix this